src/Workspaces/Core/Portable/Extensions/ValueUsageInfo.cs

```diff
@@ -13,68 +13,105 @@ internal enum ValueUsageInfo
         /// <summary>
         /// Represents default value indicating no usage.
         /// </summary>
-        None = 0x0000,
+        None = 0x0,
 
         /// <summary>
         /// Represents a value read.
         /// For example, reading the value of a local/field/parameter.
         /// </summary>
-        Read = 0x0001,
+        ValueRead = 0x1,
 
         /// <summary>
         /// Represents a value write.
         /// For example, assigning a value to a local/field/parameter.
         /// </summary>
-        Write = 0x0010,
+        ValueWrite = 0x2,
 
         /// <summary>
         /// Represents a reference being taken for the symbol.
         /// For example, passing an argument to an "in", "ref" or "out" parameter.
         /// </summary>
-        Reference = 0x0100,
+        LocationReference = 0x4,
 
         /// <summary>
         /// Represents a name-only reference that neither reads nor writes the underlying value.
         /// For example, 'nameof(x)' or reference to a symbol 'x' in a documentation comment
         /// does not read or write the underlying value stored in 'x'.
         /// </summary>
-        NameOnly = 0x1000,
+        Name = 0x8,
+
+        /// <summary>
+        /// Represents a reference to a namespace or type on the left side of a dotted name (qualified name or member access).
+        /// For example, 'NS' in <code>NS.Type x = new NS.Type();</code> or <code>NS.Type.StaticMethod();</code> or 
+        /// 'Type' in <code>Type.NestedType x = new Type.NestedType();</code> or <code>Type.StaticMethod();</code>
+        /// </summary>
+        DottedName = 0x10,
+
+        /// <summary>
+        /// Represents a generic type argument reference.
+        /// For example, 'Type' in <code>Generic{Type} x = ...;</code> or <code>class Derived : Base{Type} { }</code>
+        /// </summary>
+        GenericTypeArgument = 0x20,
+
+        /// <summary>
+        /// Represents a base type or interface reference in the base list of a named type.
+        /// For example, 'Base' in <code>class Derived : Base { }</code>.
+        /// </summary>
+        BaseTypeOrInterface = 0x40,
+
+        /// <summary>
+        /// Represents a reference to a type whose instance is being created.
+        /// For example, 'C' in <code>var x = new C();</code>, where 'C' is a named type.
+        /// </summary>
+        ObjectCreation = 0x80,
+
+        /// <summary>
+        /// Represents a reference to a namespace or type within a using or imports directive.
+        /// For example, <code>using NS;</code> or <code>using static NS.Extensions</code> or <code>using Alias = MyType</code>.
+        /// </summary>
+        NamespaceOrTypeInUsing = 0x100,
```

Comment:
IDE style is to call this "Imports" when it refers to something across VB and C# and "import" is a more general term that fits equally well with both langauges, whereas 'using' is very c# specific.

Reply:
I can change the name here.